src/modules/stake-dot/components/StakeDotAlert/StakeDotAlert.tsx

```diff
@@ -0,0 +1,34 @@
+import { Paper, Typography } from '@material-ui/core';
+import React from 'react';
+import { ReactComponent as StakeDotIcon } from '../../../features/assets/polkadot.svg';
+import { useStakeDotAlertStyles } from './StakeDotAlertStyles';
+import { t } from '../../../../common/utils/intl';
+import { useInitEffect } from '../../../../common/hooks/useInitEffect';
+
+const REFRESH_INTERVAL = 5 * 1000;
```

Comment:
```suggestion
const REFRESH_INTERVAL = 5000;
```
 I don't understand this 🤔

Reply:
This is common way to show, that it's 5 seconds. If you need 100 seconds you will not make a mistake